core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -1167,6 +1172,17 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
     // At this point the disk data either does not exist or was deleted because it failed to
     // load, so the event log needs to be replayed.
 
+    // If hybrid store is enabled, try it first.
+    if (hybridStoreEnabled) {
+      try {
+        return createHybridStore(dm, appId, attempt, metadata)
+      } catch {
+        case e: Exception =>
+          logInfo(s"Failed to create hybrid store for $appId/${attempt.info.attemptId}." +
```

Comment:
Should this be warning? may be having it as error would not be neccessary as leveldb would work?

Reply:
yeah, I think logInfo is ok since levelDB would work. And this usually occurs when the memory space is not enough.

Reply:
warn seems to be appropriate here however

Reply:
Either is fine for me personally. Let's hear more voices.